ConfigSpace/conditions.py

```diff
@@ -176,6 +209,25 @@ def evaluate(self, instantiated_hyperparameters: Hyperparameter) -> bool:
 
         return self._evaluate(evaluations)
 
+    def evaluate_vector(self, instantiated_vector: List[int]) -> bool:
+        # Then, check if all parents were passed
+        conditions = self.get_descendant_literal_conditions()
+        for condition in conditions:
+            if condition.parent_vector_id not in range(len(instantiated_vector)):
+                raise ValueError("Evaluate must be called with all "
```

Comment:
If this check isn't necessary any more, please remove it. A check whether the parent vector ID isnan would be good (hopefully doesn't take too much runtime...)